Lib/fontbakery/profiles/googlefonts.py

```diff
@@ -5308,6 +5309,39 @@ def com_google_fonts_check_stylisticset_description(ttFont):
         yield PASS, "OK"
 
 
+@check(
+    id="com.google.fonts/check/os2/fsselectionbit7",
+    rationale="""
+    All fonts should have OS/2.fsSelection bit 7 (USE_TYPO_METRICS) set \
+    so that they use typo vertical metrics instead of Win vertical metrics.
+    """,
+)
+def com_google_fonts_check_os2_fsselectionbit7(ttFonts):
+    """OS/2.fsSelection bit 7 (USE_TYPO_METRICS) is set in all fonts"""
+
+    found_fail = False
+    fail_list = []
+    for tt in ttFonts:
+        fsselection_int = tt["OS/2"].fsSelection
+        if ((fsselection_int & (1 << 7)) != 0) is True:
+            pass
+        else:
+            found_fail = True
+            fail_list.append(tt.reader.file.name)
+
+    if found_fail:
```

Comment:
Here it could be `if fail_list:` and then get rid of the `found_fail` boolean.

Reply:
TIL an empty list has truthy properties. 🤷

Reply:
eea50e5fede8aeae239e7dded7683fd0519a3edf